vs/sdl2/Makefile.minimal

```diff
@@ -1,9 +1,11 @@
 # Makefile to build the SDL library
 
-INCLUDE = -I./include
-CFLAGS  = -g -O2 $(INCLUDE)
-AR	= ar
-RANLIB	= ranlib
+INCLUDE += -I./include
```

Comment:
This should be fixed [upstream](https://github.com/libsdl-org/SDL/blob/main/Makefile.minimal).

Reply:
I can't find that file at https://github.com/libsdl-org ?

Reply:
It’s on the SDL2 branch now: https://github.com/libsdl-org/SDL/blob/SDL2/Makefile.minimal

I’ve got a PR ready for submission.

Reply:
https://github.com/libsdl-org/SDL/pull/7508 (only `CFLAGS`; the other settings can be overridden on Make’s command-line anyway).

Reply:
No they can't. If you assign a value in a makefile with `=` it will override whatever was set before or on the command line. Use `? =`.

Reply:
@roflcopter4 what version of Make are you using?

```
$ cat > Makefile <<EOF
TEXT = hello

text:
	echo "$(TEXT)"
EOF

$ make text
echo "hello"
hello

$ make text TEXT=world
echo "world"
world
```

You need to explicitly `override` variable declarations in Makefiles if you want to override values set on the command line.

Reply:
This just isn't what always happens when one uses autotools (badly). The flags are supposed to be substituted in the `configure` stage, not when running make. If that doesn't work, for any reason, then the build is broken. And it doesn't work.

Reply:
`Makefile.minimal` is only relevant if `configure` _isn’t_ used.